flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
Maybe a better way to do this is to make it consistent with the other customization hooks. Do something like

```py
class WidgetView(GenericModelView):
    api_spec = ModelApiSpec()

    def get(self, id):
        return self.retrieve(id)
```

The idea here is that it'd be nice to separate out the inheritance hierarchy of the view spec generation from that of the views themselves - that way you could have better re-use of custom spec generation across multiple unrelated views as needed.